homeassistant/components/heos/__init__.py

```diff
@@ -25,28 +26,59 @@
 
 
 async def async_setup(hass: HomeAssistantType, config: ConfigType):
-    """Set up the HEOS component."""
-    from aioheos import AioHeosController
-
+    """Create config entry from config if not already created."""
```

Comment:
I don't think we should change the docstring. We possibly start the config flow here. Creating the entry will happen at the end of the config flow. But we don't control that here.

Reply:
Reverted